posts/2019-01-14-definitive-guide-terraform-serverless.md

```diff
@@ -0,0 +1,209 @@
+---
+title: "The definitive guide to using Terraform with the Serverless Framework"
+description: "Wondering when to use Terraform and/or the Serverless Framework for your infrastructure as code? We've got answers."
+date: 2019-01-14
+thumbnail: 'https://s3-us-west-2.amazonaws.com/assets.blog.serverless.com/terraform/serverless-terraform-thumb.png'
+heroImage: 'https://s3-us-west-2.amazonaws.com/assets.blog.serverless.com/terraform/serverless-terraform-header.png'
+category:
+  - guides-and-tutorials
+authors: 
+  - SebastianBorza
+---
+
+If your organisation uses automation to manage cloud infrastructure, you’ve almost certainly heard about Terraform. And if you’ve built anything serverless, you might have noticed that deploying with the Serverless Framework is a lot like running Terraform.
+
+To which we say: you’re absolutely right. Many companies using Serverless already use Terraform, and some Serverless Framework functionality, especially when it comes to provisioning cloud resources, is similar to what Terraform can do.
```

Comment:
This sentence reads a little bit weird. Maybe something like:

`Many companies using Serverless already use Terraform, and some Serverless Framework functionality is similar to what Terraform can do, especially when it comes to provisioning cloud resources.`